ayaneo-platform.c

```diff
@@ -927,11 +966,391 @@ struct led_classdev_mc ayaneo_led_mc = {
         .subled_info = ayaneo_led_mc_subled_info,
 };
 
+/* Handling bypass charge */
+struct ayaneo_ps_priv {
+    u8 charge_types;
+    u8 current_start_threshold;
+    u8 current_end_threshold;
+    u8 bypass_available;
+};
+static struct ayaneo_ps_priv ps_priv = { POWER_SUPPLY_CHARGE_TYPE_CUSTOM, 85, 85, 0 };
```

Comment:
Should it defaults to enable bypass charge?

Reply:
No, see comments from my review.